src/app/lib/views/player/loading.js

```diff
@@ -187,7 +188,6 @@
       win.info("Loading torrent:", state);
 
       this.ui.stateTextDownload.text(i18n.__(state));
-      this.ui.stateTextStreamUrl.text(streamInfo.get('src').replace('127.0.0.1', Settings.ipAddress));
```

Comment:
I removed this line because every time I replay same movies, streamInfo.get('src') is undefined.